kotlin/legacy/legacy-workflow-rx2/src/test/java/com/squareup/workflow/legacy/rx2/WorkflowOperatorsTest.kt

```diff
@@ -77,6 +77,8 @@ class WorkflowOperatorsTest {
     states.cancel()
 
     assertThat(subscribeCount).isEqualTo(1)
-    assertThat(disposeCount).isEqualTo(1)
+    // For some reason the observable is actually disposed twice. Seems like a coroutines bug, but
+    // Disposable.dispose() is an idempotent operation so it should be fine.
```

Comment:
Interesting... I assume that's regression, worth filing Kotlin bug?

Reply:
I believe so. I'll try to isolate and file this week.